docs/integrations/custom-indicator.md

```diff
@@ -0,0 +1,78 @@
+id: custom-indicator
```

Comment:
this should solve the browsing issue (i think)
```suggestion
---
id: custom-indicator
```

Reply:
Fixed